src/images/extract.py

```diff
@@ -216,7 +216,7 @@ def replace(tag):
         elif img_src.startswith("//bits.wikimedia.org/"):
             dsk_url = img_src[46:]
 
-        elif img_src.startswith("//upload.wikimedia.org/wikipedia/es/"):
+        elif img_src.startswith("//upload.wikimedia.org/wikipedia/%s/" % config.LANGUAGE):
```

Comment:
This check may be executed millions of times so I think we should build this string only once (e.g. using a constant like `MEDIAWIKI_URL_PREFIX_LANG` at module level)